### PR TITLE
Add spell checker job in CI/CD

### PR DESCRIPTION
**Pull request by irozzo-1A**
Add spell checker job

### CODE REVIEW

File: .prow.yaml [hunk 1]
```diff
@@ -1,4 +1,20 @@
 presubmits:
+  - name: pre-kubermatic-spellcheck
```
Comment (xrstf) [inline .prow.yaml:2]: The job should be called `pre-docs-spellcheck`.

File: data/kubermatic/master/runbook.json [hunk 2]
```diff
@@ -887,7 +887,7 @@
         {
           "alert": "PromBadConfig",
           "annotations": {
-            "mesage": "Prometheus failed to reload config.",
+            "message": "Prometheus failed to reload config.",
```
Comment (xrstf) [inline data/kubermatic/master/runbook.json:890]: NB: This must be fixed upstream (Kubermatic repo), or else the next sync will overwrite this fix.

**Review by xrstf — CHANGES_REQUESTED** [review]
One change request, one hint, otherwise very nice.

File: data/kubermatic/master/runbook.json (hunk 2 above)
Comment (irozzo-1A) [reply data/kubermatic/master/runbook.json:890]: I'm going to add the same job in kubermatic repo anyway

**Review by xrstf — APPROVED** [review]
/approve

**Pull request merged**